package.json

```diff
@@ -60,7 +60,8 @@
         "prettier": "2.8.8",
         "prisma": "4.16.2",
         "semantic-release": "19.0.3",
-        "typescript": "5.1.6"
+        "typescript": "5.1.6",
+        "yaml": "^2.3.1"
```

Comment:
This `dev dependency` would have to be a `dependency` instead. I am not sure whether it's worth adding it as a dependency just to convert the JSON to yaml. It would increase the install size for all users, whether they rely on this feature or not. Also, it conflicts with the package name `prisma-json-schema-generator` a bit ;). Let's wait for some upvotes instead to figure out, whether other users would prefer to have this feature as well.